### PR TITLE
ci: Fix the error related with installing packages

### PR DESCRIPTION
**Pull request by bitboom**
This PR fixes [the CI error](https://github.com/islet-project/islet/actions/runs/8776046268/job/24079464476).

- Apply `sudo apt update` with `--fix-missing` which addressed issues caused by missing packages sources
- Fix to use `apt-get` which provides a stable CLI interface for automated environments

### CODE REVIEW

**Review by nook1208 — APPROVED** [review]
good to fix !

File: scripts/deps/pkgs.sh [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 set -e
 
-apt update && apt -y install sudo
+apt-get update && apt -y install sudo
 
-sudo apt install -y -qq --no-install-recommends \
+sudo apt-get update
```
Comment (zpzigi754) [inline scripts/deps/pkgs.sh:7]: I guess that the later `apt-get update` can be removed, as they are duplicate.

**Review by hihi-wang — APPROVED** [review]

File: scripts/deps/pkgs.sh (hunk 1 above)
Comment (bitboom) [reply scripts/deps/pkgs.sh:7]: > I guess that the one `apt-get update` can be removed, as they are duplicat

I remember this line was added to install `sudo`. Do you want to try without `apt-get update`?

Comment (zpzigi754) [reply scripts/deps/pkgs.sh:7]: Right. The first `apt-get update` would be necessary for the installing `sudo`. I mean the second `sudo apt-get update` can be removed if there is no additional purpose.

Comment (bitboom) [reply scripts/deps/pkgs.sh:7]: `apt-get update` makes below errors:

```
W: Problem unlinking the file /var/cache/apt/pkgcache.bin - RemoveCaches (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/srcpkgcache.bin - RemoveCaches (13: Permission denied)
```

Also only adding `--fix-missing` without `sudo apt-get update` couldn't resolve this. Please refer [here](https://github.com/islet-project/islet/actions/runs/8776299138)

Comment (zpzigi754) [reply scripts/deps/pkgs.sh:7]: I see. Thank you for the answer!

**Review by zpzigi754 — APPROVED** [review]

**Pull request merged**